lib/oidcstrategy.js

```diff
@@ -159,6 +203,10 @@ function Strategy(options, verify) {
   // validator will throw exception if a required option is missing
   var validator = new Validator(validatorConfiguration);
   validator.validate(itemsToValidate);
+
```

Comment:
Please have consistent line breaks after clauses. For example after ';' when validatorConfiguration is set, put a blank line.